src/components/20-molecules/top-content-bar/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@axa-ch/top-content-bar",
+  "version": "0.0.0-alpha.0",
+  "description": "Used as top of the page Warning or info box that can show text and have a CTA",
```

Comment:
I have no idea what a CTA is. Not sure if that is clear to all users.

Reply:
Should be but i will write it better

Reply:
done